### PR TITLE
Add EditorHandler + Text property

### PR DESCRIPTION
**Pull request by rachelkang**
### Description of Change ###

Implements #390 

### Additions made ###

* Adds Text to the IEditor interface
* Adds Text property map to EditorHandler
* Adds Text mapping methods to EditorHandler for Android and iOS
* Adds extension methods to apply EditorHandler Text on Android/iOS
* Adds DeviceTests for initial Text values on iOS and Android

### PR Checklist ###

- [x] Targets the correct branch 
- [x] Tests are passing (or failures are unrelated)
- [x] Targets a single property for a single control (or intertwined few properties)
- [x] Adds the property to the appropriate interface
- [x] Avoids any changes not essential to the handler property
- [x] Adds the mapping to the PropertyMapper in the handler
- [x] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [x] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [x] Tags ported renderer methods with [PortHandler]
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

### CODE REVIEW

**Comment by rachelkang** [comment]
Sample and device tests still need to be tested on iOS

File: src/Core/tests/DeviceTests/Stubs/EditorStub.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Maui.DeviceTests.Stubs
+{
+	public partial class EditorStub : StubBase, IEditor
+	{
+		public string Text { get; set; }
+
+		public Color TextColor { get; set; }
```
Comment (rachelkang) [inline src/Core/tests/DeviceTests/Stubs/EditorStub.cs:7]: TextColor is here to prevent issues wrt not implementing the full interface properly

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]
There are a couple of build errors:

`D:\agent\1\s\src\Controls\src\Core\HandlerImpl\Editor.Impl.cs(3,32): error CS0535: 'Editor' does not implement interface member 'IText.Font' [D:\agent\1\s\src\Controls\src\Core\Controls.Core.csproj]`

Could you update the `Editor.Impl.cs` definition?

File: src/Core/src/Handlers/Editor/EditorHandler.iOS.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class EditorHandler : AbstractViewHandler<IEditor, UITextView>
+	{
+		static readonly int BaseHeight = 30;
```
Comment (mattleibow) [inline src/Core/src/Handlers/Editor/EditorHandler.iOS.cs:8]: This is to match the fact that Android has the same size view and also seems Forms used the same height as well.

**Review by mattleibow — APPROVED** [review]

**Pull request merged**